### PR TITLE
refactored integration tests

### PR DESCRIPTION
**Pull request by sergueik**
Refactored integration tests  - Please feel free to advise best practices

### CODE REVIEW

File: src/test/java/com/jprotractor/integration/CommonFunctions.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		} else {
+			seleniumDriver = new PhantomJSDriver();
+		}
+		return seleniumDriver;
+	}
+
+	public static boolean checkEnvironment() {
+		Map env = System.getenv();
+
+		if (env.containsKey("TRAVIS") && env.get("TRAVIS").equals("true")) {
```
Comment (caarlos0) [inline src/test/java/com/jprotractor/integration/CommonFunctions.java:69]: Why do we do two kinds of tests?

Can't we run only the "ci" tests?

File: src/test/java/com/jprotractor/integration/CommonFunctions.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+			isCIBuild =  true;
+			isDestopTesting = false;
+		}
+		return isCIBuild;
+	}
+
+	protected static String getScriptContent(String filename) {
```
Comment (caarlos0) [inline src/test/java/com/jprotractor/integration/CommonFunctions.java]: as far as I see, you are using this function for html files, so I think it should be renamed.

File: src/test/java/com/jprotractor/integration/CommonFunctions.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+		long pollingInterval = 500;
+		WebDriverWait wait = new WebDriverWait(seleniumDriver, flexibleWait );
+		wait.pollingEvery(pollingInterval,TimeUnit.MILLISECONDS);
+		wait.until(ExpectedConditions.visibilityOf(element));
+		executeScript("arguments[0].style.border='3px solid yellow'", element);
+		Thread.sleep(highlightInterval);
+		executeScript("arguments[0].style.border=''", element);
+	}
+
+	public static void highlight(NgWebElement element, long highlightInterval) throws InterruptedException {
```
Comment (caarlos0) [inline src/test/java/com/jprotractor/integration/CommonFunctions.java]: why?

File: src/test/java/com/jprotractor/integration/NgLocalFileTest.java [hunk 4]
```diff
@@ -0,0 +1,290 @@
...
+			if (countryColumn.getText().equalsIgnoreCase("Mexico") ){
+				highlight(countryColumn);
+				cnt = cnt + 1;
+			}
+		}
+		assertTrue(cnt == 3);	
+		System.err.println("Mexico = " + cnt.toString() );
+	}		
+
+	/*
```
Comment (caarlos0) [inline src/test/java/com/jprotractor/integration/NgLocalFileTest.java]: please remove the commented out code...

**Comment by caarlos0** [comment]
@sergueik A couple of comments

File: src/test/java/com/jprotractor/integration/CommonFunctions.java (hunk 1 above)
Comment (sergueik) [inline src/test/java/com/jprotractor/integration/CommonFunctions.java:69]: IMHO it makes a lot of sense to test interactively, e.g. with Way2AutomationIntegrationTest.

**Pull request merged**